spec/integration/launcher_spec.rb

```diff
@@ -133,7 +133,7 @@
       end
     end
 
-    it_fails_firefox 'user_data_dir option should restore cookies', sinatra: true do
+    it 'user_data_dir option should restore cookies', sinatra: true do
```

Comment:
Firefox is really unstable on using user_data_dir. Sometimes working as expected, and often hangs up...